ple/games/snake.py

```diff
@@ -1,405 +1,407 @@
-import pygame
```

Comment:
What is the actual change here? Can you fix it so the _entire_ file is not being replaced